tests/test_cln_rs.py

```diff
@@ -94,7 +93,7 @@ def test_grpc_connect(node_factory):
     """Attempts to connect to the grpc interface and call getinfo"""
     # These only exist if we have rust!
 
-    grpc_port = reserve()
+    grpc_port = reserve_unused_port()
```

Comment:
Good find!  We should probably also free these up once the tests complete.  It looks like the NodeFactory class handles this already.  Maybe something like `node_factory.reserved_ports.append(grpc_port)`?  Or maybe expose reserve_unused_port as a NodeFactory method so it can track and free up the port itself?

Alternatively, `drop_unused_port()` to manually clean it up (the postgres case might need this.)

Reply:
how about now